### DISCUSSION TITLE
Envio de e-mail

### DISCUSSION THREAD

**Discussion by paulosardinha** [RamonSilva20/mapos#2617] [General]
Uso a hospedagem da hostinger e o email também é deles. Já testei com gmail, com emissão de senha app. Mas não consigo enviar e-mail automático no site hospedado na hostinger de jeito nenhum.

configurarei o cron job certinho, seguindo todas as orientações e nada.

tem alguma configuração adicional que possa fazer?

**Comment by RFS-Solutions** [comment]
Hoje percebi o mesmo aqui. Fica como pendente de envio mas nunca envia.
